Assets/Scripts/ProjectileScript.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// Script to control the projectiles once they are spawned.
+public class ProjectileScript : MonoBehaviour
+{
+    public float detonateTime = 1;
+    public float radius;
+    public float cameraShakeDuration = 0.2f;
+    public float explosionDuration = 0.2f;
+
+    public GameObject explosionFx;
+
+    SpriteRenderer spriteRenderer;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        spriteRenderer = GetComponent<SpriteRenderer>();
+
+        StartCoroutine(TickBomb());
+    }
+
+    // Function to make the bomb flicker white and red 
+    IEnumerator TickBomb()
+    {
+        Invoke("Detonate", detonateTime);
+        while (true)
+        {
+            spriteRenderer.color = Color.red;
+            yield return new WaitForSeconds(.1f);
+            spriteRenderer.color = Color.white;
+            yield return new WaitForSeconds(.1f);
+        }
+    }
+
+    // Makes calls to explode the bomb, spawn an explosion and shake the camera.
+    void Detonate()
+    {
+        TerrainDestroyer.instance.DestroyTerrain(transform.position, radius);
+
+        SpawnExplosionFX();
+        DoCameraShake();
+
+        Destroy(gameObject);
+    }
```

Comment:
Again, probably out of the scope of this game jam... but probably it would be better to send an event here, and then pick that event by the other modules (destroying the terrain, doing the camera shake, etc...).

Reply:
That's a good point.  The Event system would probably make this way simpler